fe/fe-core/src/main/java/com/starrocks/scheduler/persist/DropTasksLog.java

```diff
@@ -0,0 +1,37 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.scheduler.persist;
+
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.common.io.Text;
+import com.starrocks.common.io.Writable;
+import com.starrocks.persist.gson.GsonUtils;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.List;
+
+public class DropTasksLog implements Writable {
+
+    @SerializedName("taskNameList")
+    List<String> taskNameList;
```

Comment:
Why not use taskId?
If taskA is dropped, and create taskA again, they have the same name, however they have the different ids.